src/pages/Address/BTCAddressComp.tsx

```diff
@@ -103,6 +112,26 @@ export const BTCAddressOverviewCard: FC<{ address: Address }> = ({ address }) =>
                 </div>
               </AddressAssetsTabPane>
             ) : null}
+            {hasInvalid ? (
+              <AddressAssetsTabPane
+                tab={
+                  <AddressAssetsTabPaneTitle onClick={() => setActiveTab(AssetInfo.Invalid)}>
+                    {t('address.invalid')}
+                  </AddressAssetsTabPaneTitle>
+                }
+                key={AssetInfo.Invalid}
+              >
+                <div className={styles.assetCardList}>
+                  <RgbppAssets
+                    address={address.bitcoinAddressHash}
+                    count={+address.liveCellsCount}
+                    udts={udts}
+                    isUnBounded
+                    inscriptions={inscriptions}
```

Comment:
Will all `inscription`s be displayed in the `invalid` panel?

Reply:
> Will all `inscription`s be displayed in the `invalid` panel?

No, but @Sven-TBD said that assets would be displayed here in the future, so I just left them here

Reply:
> Inscriptions could be displayed if it is invalid. Actually , it doesn't matter which kind type of asset it is , as long as it is invalid, it will be displayed here.

By the reason which was talked about before in the channel, assets could not be displayed in this PR.

Reply:
So `invalid inscriptions` are expected to be here, but the `inscriptions` in the code are not grouped into `valid` and `invalid`, am I right @Daryl-L

Reply:
> > So `invalid inscriptions` are expected to be here, but the `inscriptions` in the code are not grouped into `valid` and `invalid`, am I right @Daryl-L
> 
> Yes, including the UDTs

Now there are 2 solutions:
1. Skip displaying invalid inscriptions for now; Or
2. Update the API to return invalid inscriptions

Am I right? How will we move forward?

Reply:
I think better to skip displaying invalid inscriptions for now, and
Update the API to return invalid inscriptions in the future.
@Sven-TBD

Reply:
So the `inscriptions` should be removed if invalid inscriptions are not ready for now, am I right @Daryl-L

Reply:
> So the `inscriptions` should be removed if invalid inscriptions are not ready for now, am I right @Daryl-L

I think it is not necessary. Here I use the `RgbAsset` component, about three components would be modified if I remove it, and it would be added back in the future. What about just left it and add a "TODO".

Reply:
> > So the `inscriptions` should be removed if invalid inscriptions are not ready for now, am I right @Daryl-L
> 
> I think it is not necessary. Here I use the `RgbAsset` component, about three components would be modified if I remove it, and it would be added back in the future. What about just left it and add a "TODO".

Sure, it can be marked `TODO`, but the `valid inscription` will be listed in the `hasInvalid` branch. An empty list should be passed in.